### DISCUSSION TITLE
Lists

### DISCUSSION THREAD

**Discussion by jmutnick** [alandtse/alexa_media_player#2102] [Ideas]
Now that Home Assistant supports to do list integrations, any thought on integrating with Alexa's Lists?

**Comment by finipini** [comment]
You already can do that. In this release we can have too the lists in Todoist , and Todoist is a skill in Alexa that automaticaly grabs the lists in Alexa . So Todoist acts hás the middle men between HA and Alexa.

It works very well btw.

https://www.home-assistant.io/integrations/todoist/


I use it ONLY for shopping list, but HA grabs all the lists.

**Comment by clyra** [comment]
It would be nice to no rely on another third party service (todoist) just to sync your lists. Alexa has a API for that (https://developer.amazon.com/en-US/docs/alexa/list-skills/list-management-api-reference.html). Using the "ask-cli" to generate a token you can easily grab lists ids using curl and the "api.amazonalexa.com//v2/householdlists/" or the list itself using /v2/householdlists/<list_id>. Since alexa_media_player already do the the authentication and refresh token stuff, how about to also query the lists and present them as attributes as it's already done with devices status, last_called, etc?

**Comment by alexiri** [comment]
Amazon has killed the List Skills API. Is there another way to get list data?

**Comment by PlayedIn** [comment]
Yeah Amazon killed the Todoist solution.   Anyone have a work-around to get Amazon lists into HA?
